modules/apps/calendar/calendar-test/src/testIntegration/java/com/liferay/calendar/service/test/CalendarBookingLocalServiceTest.java

```diff
@@ -3434,6 +3427,40 @@ private List<WorkflowTask> _getWorkflowTasks() throws Exception {
 		return workflowTasks;
 	}
 
+	private void _testAddCalendarBookingWithVideoDescription(
+			String expectedValues)
```

Comment:
@fabioafreitas this isn't a common parameter name on the portal, and doesn't fit very well on this logic. Could you came up with other options to me? Or just define as `html` as was before.